integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/SetConfigurationObject.feature

```diff
@@ -72,17 +98,21 @@ Feature: SmartMetering Configuration
       | ConfigurationFlagEnabled_9  | false                  |
       | ConfigurationFlagType_10    | HLS_5_ON_PO_ENABLE     |
       | ConfigurationFlagEnabled_10 | false                  |
+    Examples:
+      | deviceIdentification |
+      | TEST1024000000001    |
+      | TEST1027000000001    |
```

Comment:
Should we test other protocol versions as well? For example, what should a DSMR2.2 meter do?

Reply:
Added cucumber tests for all (D)SMR protocol versions

Reply:
DSMR 2.2 is now tested to fail